src/service/DatasetEditManager.ts

```diff
@@ -55,14 +57,22 @@ export class DatasetEditManager {
     ) {
         subscriptions.push(
             vscode.commands.registerCommand(
-                "zosexplorer.editMember",
+                "zosexplorer.edit",
                 async (arg) => {
+                    let passedMember: Member = { name: ""};
+                    const DSO: string = arg.dataset.dataSetOrganization;
+                    if (DSO === DSOrg.PO) {
+                        passedMember = arg.member;
+                    } else {
+                        passedMember.name = arg.dataset.name;
+                    }
                     try {
                         await this.editMember(
                             arg.host,
                             arg.dataset,
-                            arg.member,
+                            passedMember,
                             dataProvider,
+                            DSO,
```

Comment:
Why `editMember` function should depend on DSO?
Please consider special method for dataset editing.

Reply:
method sign reverted to the original.